test/e2e/alertmanager_test.go

```diff
@@ -367,7 +367,7 @@ route:
   repeat_interval: 12h
   receiver: 'webhook'
 receivers:
-- name: 'webhook'
+- name: 'firstConfigWebHook'
```

Comment:
the test checks that the `firstConfigWebHook` string appears in the config dumped  from the Alertmanager API but the webhook URL is now a secret field and is masked.